### ISSUE TITLE
iOS Accessibility - Master ticket (has many in it)

### ISSUE THREAD

**Issue by RaulConstan** [bcgov/bc-wallet-mobile#3221]
The following are accessibility issues found:
1. twin number 1

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/78af0226-2939-460d-8ed3-7733db889a36" />

2. Unnecessary commands

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/41a9c701-b57b-4317-84bd-92c0da5924ad" />
 
3. Unnecessary command

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/4e275b5f-074c-4ade-8918-02ab45a5f24e" />

5. two screens, activating the second screen kicks user out of commands (Blocker)

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/d541d600-526f-4521-a341-ab64912c0734" />

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/16ca4a10-ad44-4533-9aaf-0902e40cfebb" />

6. Poor positioning, also controls do not work! (Blocker)

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/2948459b-a56e-458e-8e6f-bd648811dbf4" />

7. Timeout from DoB rollers screen

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/adf3321c-9013-477d-a57d-d0b37e18f6d5" />

8. Missing commands

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/4f56cd3a-c5df-4505-8351-9a8ce1cafe40" />

9. Missing command

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/e3a566ab-d402-457c-8002-df507e86106d" />

10. Missing commands

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/d0d0755f-781b-4159-acfb-e59b692808fa" />

11. Missing command + misalignments

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/61bb9fc4-9354-4c27-9313-b01e76aa8d1a" />

12. Missing command

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/9fe37913-b9eb-433c-bf0c-a78e3fa1be7e" />

13. Missing command

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/7811b735-0f8e-4918-9e58-9adcbcab2446" />

14. Missing Command

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/a2a0ea65-0f61-4494-b5aa-7c7bd02a996f" />

15. Missing Command & Superfluous others...

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/ebdde3ec-f803-4df2-90c9-83f65520e836" />

16. Cannot enter code without first hitting the unhide

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/233821b7-b8d4-43ab-bdc8-453bb0039e71" />

17. Missing command

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/d05ceec7-d7c5-4d22-a755-359280cee803" />

18. Missing command

<img width="462" height="1128" alt="Image" src="https://github.com/user-attachments/assets/9eacaf6a-1091-48ec-9b69-ca47d00482ab" />

**Comment by jeznorth** [comment]
Moving back to refinement for sizing.

**Comment by jleach** [comment]
Fixed in 4.0.0-5636.

**Comment by RaulConstan** [comment]
Confirmed in 5708.
